buildbot.mariadb.org/master-libvirt/master.cfg

```diff
@@ -206,19 +165,12 @@ f_deb_install.addStep(getScript('deb-galera.sh'))
 f_deb_install.addStep(getDebGaleraStep("2223"))
 f_deb_install.addStep(steps.ShellCommand(name="cleanup", command="rm -r * .* 2> /dev/null || true", alwaysRun=True))
 
-## f_deb_major_upgrade
-f_deb_major_upgrade = util.BuildFactory()
-f_deb_major_upgrade.addStep(steps.SetPropertyFromCommand(name="major_version", property="major_version", command=util.Interpolate("sh -c \"echo '%(prop:branch)s' | sed -e \\\"s/.*\\\\(5\\\\.5\\\\|10\\\\.[0-9]\\\\).*/\\\\1/\\\"\"")))
-f_deb_major_upgrade.addStep(getScript('deb-upgrade.sh'))
-f_deb_major_upgrade.addStep(getDebMajorUpgradeStep())
-f_deb_major_upgrade.addStep(steps.ShellCommand(name="cleanup", command="rm -r * .* 2> /dev/null || true", alwaysRun=True))
-
-## f_deb_minor_upgrade
-f_deb_minor_upgrade = util.BuildFactory()
-f_deb_minor_upgrade.addStep(steps.SetPropertyFromCommand(name="major_version", property="major_version", command=util.Interpolate("sh -c \"echo '%(prop:branch)s' | sed -e \\\"s/.*\\\\(5\\\\.5\\\\|10\\\\.[0-9]\\\\).*/\\\\1/\\\"\"")))
-f_deb_minor_upgrade.addStep(getScript('deb-upgrade.sh'))
-f_deb_minor_upgrade.addStep(getDebMinorUpgradeStep())
-f_deb_minor_upgrade.addStep(steps.ShellCommand(name="cleanup", command="rm -r * .* 2> /dev/null || true", alwaysRun=True))
+## f_deb_upgrade
+f_deb_upgrade = util.BuildFactory()
+f_deb_upgrade.addStep(steps.SetPropertyFromCommand(name="major_version", property="major_version", command=util.Interpolate("sh -c \"echo '%(prop:branch)s' | sed -e \\\"s/.*\\\\(5\\\\.5\\\\|10\\\\.[0-9]\\\\).*/\\\\1/\\\"\"")))
+f_deb_upgrade.addStep(getScript('deb-upgrade.sh'))
+f_deb_upgrade.addStep(getDebUpgradeStep())
+f_deb_upgrade.addStep(steps.ShellCommand(name="cleanup", command="rm -r * .* 2> /dev/null || true", alwaysRun=True))
```

Comment:
clean is not needed

Reply:
@cvicentiu yes, that is the case. Also, for install/upgrade simply removing the files is not enough anyway, so that can be skipped

Reply:
Exact, this is not so impacting performant wise, but it's confusing an can make think that we do not start on a clean instance (same goes for containers BTW). We do not let time for BB to re-use any worker instance. So no cleaning step is needed at all.